package.json

```diff
@@ -7,8 +7,15 @@
     "test": "tests"
   },
   "scripts": {
+    "setup": "npm run build",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "theme": "cd ./web/themes/custom/karaoke && yarn start"
+    "theme-build": "cd ./web/themes/custom/kitchen_karaoke && npm install && npm run build",
+    "theme": "cd ./web/themes/custom/kitchen_karaoke && npm run develop",
+    "build": "lando start && npm run get-db && lando composer install && lando drush updb -y && npm run theme-build && lando drush cr && lando drush uli ",
+    "get-db": "lando pull --code=none --files=none --database=pr-5",
```

Comment:
Wait... what?
Shouldn't this be a pull from live if we are using the lando pull command?

Reply:
I set it to this to make it smoother for those setting it up. It works fine with pulling from live. I'll change it to that. It isn't that it doesn't work, just that I figured we may end up working from this branch for a bit. I'll correct this.